js-src/aws/s3Upload.ts

```diff
@@ -17,32 +26,25 @@ export const uploadFiles = (files: FileList) => {
             Logins: {
                 [cognitoLoginId]: session.getIdToken().getJwtToken()
             }
-        });
-
-        const bucket = "tdr-files";
+        });        
+       
+        const userName = currentUser.getUsername();
+        const bucket = S3_UPLOAD_BUCKET;
+        const s3Uploadkey = `${userName}-` + uuidv4();
```

Comment:
Just FYI (don't bother changing anything at this point), the folder will probably end up being the user's identity pool ID rather than their user name.

See https://github.com/nationalarchives/tdr-dev-documentation/blob/master/technology-considerations/user-management-in-cognito.md#authorization-for-file-uploads for more context.